src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,12 @@
+#databaseChangeLog: []
+databaseChangeLog:
+  - include:
+      file: "/db/changelog/changes/1-create-info-phoneNumber-table-changelog.sql"
```

Comment:
can you use native liquibase commands like these [here](https://docs.liquibase.com/change-types/create-table.html?Highlight=create%20table)

We should avoid using direct sql files whenever possible since this could potentially need rewrite if we switch to another DB

Reply:
Which format should be used?